app/lib/validation.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash')
+
+const addEntityIfValid = function (object, params, modelName, name) {
+  try {
+    object[name] = Reflect.construct(modelName, [params]).basicObject()
```

Comment:
This is straight JS magic 👌 . We can add more structure/ comments if this pattern is used more in the future